server/index.js

```diff
@@ -0,0 +1,249 @@
+require('dotenv').config();
```

Comment:
This file should not be in this PR. Find the commit that caused this and clean it up, please!

Reply:
Not sure how this got here; is this because of me merging w master?